apps/dg/components/case_card/case_card.js

```diff
@@ -22,6 +22,14 @@ DG.React.ready(function () {
             // leave some room for collection name (left) and navigation buttons (right)
             kMinColumnWidth = 82;
 
+        var kThresholdDistance2 = 0, // pixels^2
+            tTouchID,
+            tTouchTimer,
+            tPointerIsDown = false,
+            tStartCoordinates,
+            tDragInProgress = false,
+            tDragHandler;
```

Comment:
These variables were originally defined inside the `renderAttribute()` function along with the event handling functions that use them. This meant that 1) each attribute had its own copy of them and 2) every time a re-render occurred a new set of them was generated along with a new set of event handler functions. Moving them to the scope of the case card function closure makes them effectively global.